Dockerfile-cli

```diff
@@ -40,6 +40,9 @@ COPY src/gpg /usr/local/etc/gpg
 COPY src/php/conf/ /usr/local/etc/php/conf.d/
 COPY src/php/cli/conf/*.ini /usr/local/etc/php/conf.d/
 
+# Install curl-dev in order to address the curl binary issue in some Alpine versions
+RUN apk add --no-cache curl-dev
```

Comment:
It would be slightly better to add this to one of the other `apk add` invocations instead, to avoid adding another layer to the image.

Reply:
was about to ask why the differnce in approaches and suggest moving it up with the apk adds in the block above